conda_build_config.yaml

```diff
@@ -232,21 +132,662 @@ cross_compiler_target_platform:
 target_platform:
   - win-64                     # [win]
   # - win-32                     # [win]
-tk:
-  - 8.6
-vc:
-  - 14                         # [win]
-zlib:
-  - 1.2
-xz:
-  - 5
 channel_targets:
   - defaults
 cdt_name:
   - amzn2    # [linux and aarch64]
-zip_keys:
-  -
-    - python
-    - numpy
+
+
+## GLOBAL pinnings
+alsa_lib:
+  - '1.2'
+ampl_mp:
+  - '3.1'
+aom:
+  - '3.6'
+armadillo:
+  - '12'
+arpack:
+  - '3.9'
+arrow_cpp:
+  - 19.0.0
+assimp:
+  - 5.4.3
+at_spi2_core:
+  - '2.36'
+atk_1.0:
+  - 2.36.0
+aws_c_auth:
+  - 0.8.5
+aws_c_cal:
+  - 0.8.5
+aws_c_common:
+  - 0.11.1
+aws_c_compression:
+  - 0.3.1
+aws_c_event_stream:
+  - 0.5.4
+aws_c_http:
+  - 0.9.3
+aws_c_io:
+  - 0.17.0
+aws_c_mqtt:
+  - 0.12.2
+aws_c_s3:
+  - 0.7.11
+aws_c_sdkutils:
+  - 0.2.3
+aws_checksums:
+  - 0.2.3
+aws_crt_cpp:
+  - 0.31.0
+aws_sdk_cpp:
+  - 1.11.528
+azure_core_cpp:
+  - 1.14.1
+azure_identity_cpp:
+  - 1.10.1
+azure_storage_blobs_cpp:
+  - 12.13.0
+azure_storage_common_cpp:
+  - 12.10.0
+backtrace:
+  - '20241216'
+blosc:
+  - '1'
+boost:
+  - 1.82
+boost_cpp:
+  - 1.82
+brunsli:
+  - '0'
+bzip2:
+  - '1'
+c_ares:
+  - '1'
+c_blosc2:
+  - '2'
+cairo:
+  - '1'
+capnproto:
+  - 1.1.0
+ceres_solver:
+  - '2.2'
+cfitsio:
+  - '3.470'
+charls:
+  - '2.2'
+coin_or_cbc:
+  - '2.10'
+coin_or_cgl:
+  - '0.60'
+coin_or_clp:
+  - '1.17'
+coin_or_osi:
+  - '0.108'
+coin_or_utils:
+  - '2.11'
+cudnn:
+  - '9'
+cutensor:
+  - '2'
+cyrus_sasl:
+  - 2.1.28
+dav1d:
+  - 1.2.1
+dbus:
+  - '1'
+eigen:
+  - 3.3.7
+epoxy:
+  - '1.5'
```

Comment:
Sometimes, there are numbers `- 1.2.1` but sometimes strings with a single quote `- '1.5'`. It should be standardized.

Reply:
It is standardized in a way. 1.2.1 is obviously a string so set that way by ruamel. 1.5 is ambiguous so is quoted by ruamel.